### DISCUSSION TITLE
월드상품에 출시되어 있는 상품 등록 가능 여부

### DISCUSSION THREAD

**Discussion by LeeYongSu1** [naverz/zepeto-studio-kor#431] [5. Q&A]
유니티로 제작한 월드에서 제페토에서 판매중인 아이템을 판매할 수 있나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 월드 내에서는 나만의 고유한 아이템을 판매할 수 있습니다. 
[아이템 판매 가이드 문서](https://studio.zepeto.me/guides/in-world-purchases)
다른 분이 만든 아이템은 저작권 이슈로 판매가 불가한 점 이해 부탁 드립니다.

**Reply by LeeYongSu1** [reply]
본인 계정에 출시 되어 있는 아이템이더라도 판매가 불가능한가요?
월드 상품 등록에 현재 본인이 출시한 상품을 등록하는 칸은 없고, 월드 상품 만들기만 있어서요

**Reply by Zepeto-TechEvangelist** [reply]
네, 기존에 출시된 아이템은 불가하며 월드를 개발하면서 새로 만든 상품만 판매 가능합니다.
